packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -68,15 +68,27 @@ export default class QuanticSearchBoxSuggestionsList extends LightningElement {
 
   /** @type {number} */
   @track selectionIndex = -1;
+  /** @type {boolean} */
+  initialRender = true;
```

Comment:
It's a detail but typically the default value of a boolean is false, so to do only 1 render, usually the pattern I've seen is just you don't declare a `hasRendered` variable and then in the renderedCallback

```
if(!this.hasRendered) {
  [...]
  this.hasRendered = true;
}
```